packages/drivers/local-driver/api-report/local-driver.legacy.alpha.api.md

```diff
@@ -4,6 +4,8 @@
 
 ```ts
 
+/// <reference types="node" />
+
```

Comment:
This does not look preferred. Don't we have uses against local-driver that are not Node.js?

Reply:
@ChumpChief do you know the answer to this? I haven't worked with this much, and the README is empty (aside from our standard template disclaimers).